packages/common/src/core/schemas/internal/metrics/ProjectUiSchemaMetrics.ts

```diff
@@ -60,12 +63,65 @@ export const buildUiSchema = (
               },
             },
           },
+          {
+            scope: "/properties/_metric/properties/valueType",
+            type: "Control",
+            labelKey: `${i18nScope}.fields.metrics.valueType.label`,
+            rule: SHOW_FOR_STATIC_RULE_ENTITY,
+            options: {
+              control: "hub-field-input-tile-select",
+              layout: "horizontal",
+              helperText: {
+                labelKey: `${i18nScope}.fields.metrics.valueType.helperText`,
+                placement: "bottom",
+              },
+              enum: {
+                i18nScope: `${i18nScope}.fields.metrics.valueType.enum`,
+              },
+            },
+          },
           {
             labelKey: `${i18nScope}.fields.metrics.value.label`,
             scope: "/properties/_metric/properties/value",
             type: "Control",
-            rule: SHOW_FOR_STATIC_RULE_ENTITY,
+            rule: SHOW_FOR_STATIC_AND_STRING_RULE_ENTITY,
+            options: {
+              messages: [
+                {
+                  type: "ERROR",
+                  keyword: "minLength",
+                  labelKey: `${i18nScope}.fields.metrics.value.message.minLength`,
+                  icon: true,
+                  allowShowBeforeInteract: true,
+                },
+              ],
+            },
+          },
+          {
+            labelKey: `${i18nScope}.fields.metrics.value.label`,
+            scope: "/properties/_metric/properties/value",
+            type: "Control",
+            rule: SHOW_FOR_STATIC_AND_NUMBER_RULE_ENTITY,
+            options: {
+              type: "number",
```

Comment:
I know this is already how it's implemented in the `input` field, but I don't love that we need to specify the `type` in the `uiSchema.options`. It feels like this should come directly from the `schema.type` which already specifies that this is a `number`. Originally I was thinking we could change the [`type` getter](https://github.com/ArcGIS/opendata-ui/blob/master/packages/hub-components/src/components/arcgis-configuration-editor/components/arcgis-configuration-editor-field/fields/input/input.tsx/#L35-L45) in the input field to something like: 

```tsx
get type(): any {
  const shemaTypeToType = {
    integer: 'number',
    number: 'number',
    string: 'text'
  };
  const schemaFormatToType = {
    ['date-time']: 'datetime-local',
    ['idn-email']: 'email'
  }

  return Array.isArray(this.params.schema.type)
    ? "text"
    : schemaFormatToType[this.params.schema.format]
      || this.params.schema.format
      || this.params.uiSchema?.options?.type
      || shemaTypeToType[this.params.schema.type]
}
```

But I think we'd actually run into issues when using conditional subschemas like what I've proposed above because the `type` isn't actually stored in the property's schema 🤔

So anyways, this is just a long way of saying stick with this for now 😅